pkgs/tools/system/mlc/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, patchelf, lib }:
```

```suggestion
{ stdenv, fetchurl, patchelf }:
```

You can use ``with stdenv;``.

---

Done.